hooks/tk-multi-loader2/tk-katana_actions.py

```diff
@@ -136,26 +136,34 @@ def execute_action(self, name, params, sg_publish_data):
         # resolve path
         path = self.get_publish_path(sg_publish_data)
 
-        if name == "open_project":
-            self._open_project(path, sg_publish_data)
+        if name == "import_project":
+            self._import_project(path, sg_publish_data)
 
         if name == "import_look_file":
-            self._open_project(path, sg_publish_data)
-
-        if name == "create_node_Alembic_In":
+            node = self._create_node(
+                "LookFileAssign",
+                path, 
+                sg_publish_data,
+                path_parameter="args.lookfile.asset.value"
+            )
+            node.getParameter('args.lookfile.asset.enable').setValue(1.0, 0)
+
+        if name == "import_alembic":
             self._create_node("Alembic_In", path, sg_publish_data, path_parameter="abcAsset")
 
-        if name == "create_node_ImageRead":
+        if name == "import_image":
             self._create_node("ImageRead", path, sg_publish_data, path_parameter="file")
 
 
     ##############################################################################################################
     # helper methods which can be subclassed in custom hooks to fine tune the behaviour of things
 
-    def _open_project(self, path, sg_publish_data):
+    def _import_project(self, path, sg_publish_data):
```

Comment:
Add args for path. Maybe we can get rid of `sg_publish_data` as it's unused.

```suggestion
    @staticmethod
    def _import_project(path):
```